src/app/models/form.server.model.ts

```diff
@@ -463,15 +503,7 @@ const compileFormModel = (db: Mongoose): IFormModel => {
     this: IFormModel,
     formId: string,
   ): Promise<IPopulatedForm | null> {
-    return this.findById(formId).populate({
-      path: 'admin',
-      // Remove irrelevant keys from populated fields of form admin and agency.
-      select: '-__v -created -lastModified -updatedAt -lastAccessed',
-      populate: {
-        path: 'agency',
-        select: '-__v -created -lastModified -updatedAt',
-      },
-    })
+    return this.findById(formId).populate(POPULATE_FORM_PARAMS)
```

Comment:
I checked the use of `getFullFormById` and it's only used for internal server-side functions; I think we can simplify the code to= disregard the public view of a form:

```suggestion
    return this.findById(formId).populate({
      path: 'admin',
      populate: {
        path: 'agency',
      },
    }))
```

Reply:
Good suggestion, we can always use `getPublicView` instance method to pick it out if we need to yea?

Reply:
yeah, and i think we can lose the `getFormPublicView` service method after that